apps/console/src/features/identity-providers/components/settings/attribute-management/attribute-mapping-list-item.tsx

```diff
@@ -0,0 +1,301 @@
+import { FormApi } from "final-form";
```

Comment:
Add proper licensing header.

Reply:
Thanks! I've added missing license headers for all the files. Can be find in commit b49a285